docs/public/upgrading/backend/upgrade_zonemaster_backend_ver_11.2.0.md

```diff
@@ -0,0 +1,17 @@
+# Upgrade to 11.2.0
```

Comment:
That might be my fault, because I introduced a migration script in https://github.com/zonemaster/zonemaster-backend/pull/1166 assuming that the next version number would be 11.2.0. That script should be renamed in a follow-up PR.